server/src/index.js

```diff
@@ -23,26 +24,29 @@ app.use(compression());
 app.use(helmet());
 
 // cors enable
-app.options('*', cors());
-app.use(cors({ origin: 'http://localhost:5000' }));
+app.options("*", cors());
+app.use(cors({ origin: "http://localhost:5000" }));
 
 // body-parser
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json());
 
 // database connection
-mongoose.connect(process.env.MONGO_URL,
+mongoose
+  .connect(
+    `mongodb+srv://${process.env.MONGO_DB_USER}:${process.env.MONGO_DB_PASSWORD}@kurakoo.4ebj8.mongodb.net/${process.env.MONGO_DB_DATABASE}?retryWrites=true&w=majority`,
```

Comment:
The rest in the db setup is fine but can leave it as before because the setup you introduced is for online cluster only, which is not every developer adopts as when you use local db then there isn't any username and password required that is why `mongo_url` is sufficient. So if you wanna use atlas clusters, go ahead but just add the complete url with creds onto your `.env` file

Reply:
Kindly don't unresolve it yourself, as I will do that after you have pushed code

Reply:
If you mentioned above that don't delete the files then why are you, again and again, commented on each part of the deleted part, I got your point from the above-mentioned comment :)